pkg/addon/utils.go

```diff
@@ -409,3 +439,21 @@ func generateAnnotation(meta *Meta) map[string]string {
 func isErrorCueRenderPathNotFound(err error, path string) bool {
 	return err.Error() == fmt.Sprintf("var(path=%s) not exist", path)
 }
+
+func checkConflictDefs(ctx context.Context, k8sClient client.Client, defs []*unstructured.Unstructured, app v1beta1.Application) ([]string, error) {
```

Comment:
```suggestion
func checkConflictDefs(ctx context.Context, k8sClient client.Client, defs []*unstructured.Unstructured, appName string) ([]string, error) {
```

Reply:
Maybe we need use  other information（annotations or labels）about this app in the future？

Reply:
please  follow the KISS (keep it simple and stupid) principle